ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiPartyManagementService.scala

```diff
@@ -161,6 +161,12 @@ private[apiserver] final class ApiPartyManagementService private (
         metadataO = recordO.map(_.metadata),
         recordO.map(_.identityProviderId),
       )
+    case (details, _) if identityProviderId == IdentityProviderId.Default =>
```

Comment:
maybe a comment would be needed to explain the why

Reply:
added a comment